Gemfile

```diff
@@ -100,7 +104,7 @@ end
 
 group :development, :test do
   # Use RSpec for Behaviour testing
-  gem 'rspec-rails'
+  gem 'rspec-rails', '6.1.2'
```

Comment:
why lock this version?

Reply:
According to its [gem repo](https://github.com/rspec/rspec-rails), specific Rails version requires specific Rspec version as well, and according to their policy, version 6.1 and 7.0 should be kept at Rspec 6.1 (though, it's not sure about their plan for Rails 7.1, whether it's going to be RSpec 6.x or Rspec 7.x), that's why I locked it to 6.1.

The newest version was just released less than a month ago (they have released it previously for 6.1.3 but they got bug and they just announced to rollback to 6.1.2. I tried to search for that announcements but I couldn't find it anymore)

Perhaps I should lock it to 6.x